source/administration/command-line-tools.rst

```diff
@@ -106,6 +106,7 @@ mattermost
     .. code-block:: none
 
       -c, --config {string}   Configuration file to use. (default "config.json")
+      -- disableconfigwatch {boolean}   When set config.json will not be loaded from disk when the file is changed (default "false")
```

Comment:
Not sure I understand - is it that if it's disabled, config.json will not be loaded from the disk when the JSON file is updated?

Reply:
@lieut-data can you please help me with this question?

Reply:
First, a minor spacing change:
```suggestion
      --disableconfigwatch {boolean}   When set config.json will not be loaded from disk when the file is changed (default "false")
```

Second, this comes right out of the `mattermost CLI` help, so if we change, we should probably do it in both places. Here's a different stab at the help:
```suggestion
      --disableconfigwatch {boolean}   When set, config.json will not be reloaded automatically when another process changes it (default "false")
```

Downside it's a bit long, and already the longest help item.